ctapipe/reco/FitGammaHillas.py

```diff
@@ -0,0 +1,308 @@
+from itertools import combinations,permutations
+
+import numpy as np
+
+from scipy.optimize import minimize
+
+from astropy import units as u
+u.dimless = u.dimensionless_unscaled
+
+from ctapipe.utils import linalg
+
+from ctapipe.reco.reco_algorithms import RecoShowerGeomAlgorithm
+from ctapipe.io.containers import RecoShowerGeom
+
+
+
+__all__ = ["FitGammaHillas"]
+
+
+
+
+def guessPixDirection(pix_x, pix_y, tel_phi, tel_theta, tel_foclen, camera_rotation=-100.893 * u.degree):
+    # TODO replace with proper implementation
+    # beta is the pixel's angular distance to the centre according to beta / tel_view = r / maxR
+    # alpha is the polar angle between the y-axis and the pixel
+    # to find the direction the pixel is looking at, 
+    #  - the pixel direction is set to the telescope direction
+    #  - offset by beta towards up
+    #  - rotated around the telescope direction by the angle alpha
+    
+    pix_alpha = np.arctan2(pix_x, pix_y)
+    pix_beta  = (pix_x**2 + pix_y**2)**.5
+
+    pix_beta  = pix_beta / tel_foclen * u.rad 
```

Comment:
also concerning all other comments to this function:
This is just a temporary solution @ParsonsRD is working on a proper implementation of coordinate transformations. This function is only used here and not intended for further spread. It will be removed as soon as dan's transformation from the camera frame to alt_az is available.